### DISCUSSION TITLE
Early Pre-Release: Nightly 1 (r3_8_2nightly1) has been released

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#2484] [Announcements]
Since the JSON-RPC feature has been merged, I've tagged a nightly. This can and should be tested for everyone who wants to test "bleeding edge". 
There have been quite some other smaller improvements, so it would be great to have people testing it. Note: There might be stability issues or unknown behaviour. This is an EARLY PRERELEASE.

Please see the [changelog](https://github.com/jamulussoftware/jamulus/releases/tag/r3_8_2nightly1) for more information.

## Downloads

_Windows users: Please note that SmartScreen will probably display warnings about this release being unknown upon download and/or execution of the installer._

**[↓ Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_3.8.2dev-b90741e_win.exe)** (ASIO version), alternative: [↓ JACK version](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_3.8.2dev-b90741e_win_jack.exe)
**[↓ macOS](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_3.8.2dev-b90741e_mac.dmg)** for High Sierra (10.13) to Big Sur (11) (unsigned) and [↓ macOS legacy build](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_3.8.2dev-b90741e_mac_legacy.dmg) (unsigned) for macOS Sierra (10.12), El Capitan (10.11) or Yosemite (10.10).
**[↓ Debian/Ubuntu (amd64)](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_3.8.2dev-b90741e_ubuntu_amd64.deb)**, alternative: [↓ headless version](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_headless_3.8.2dev-b90741e_ubuntu_amd64.deb)
**[↓ Android](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_3.8.2dev-b90741e_android.apk)** (experimental)
**[↓ iOS](https://github.com/jamulussoftware/jamulus/releases/download/r3_8_2nightly1/jamulus_3.8.2dev-b90741e_iOSUnsigned.ipa)** (experimental. Unsigned: Needs to be signed before installation on device. Please see the [iOS install page](https://github.com/jamulussoftware/jamuluswebsite/blob/b1468429dae22e348baf8a6aed452e90d15c0de8/wiki/en/Installation-for-iOS.md))

**Comment by dtinth** [comment]
My servers has been upgraded to be based on this version. Apart from the newly-added API authentication mechanism, the upgrade went smoothly.

Some extra patches are applied to my servers to provide additional APIs: https://github.com/dtinth/jamulus/compare/dtinth_base...dtinth
